lib/smart_answer_flows/coronavirus-employee-risk-assessment/outcomes/go_back_to_work.govspeak.erb

```diff
@@ -1,9 +1,56 @@
+<% render_content_for :title do %>
+  You can go back to work
+<% end %>
+
 <% render_content_for :body do %>
-  $CTA
-    ## You can go back to work
-    You can go back into work, but your employer must make arrangements for you to work safely.
 
-    [Check the guidance on working safely, and what you will need to do](https://www.gov.uk/coronavirus-business-reopening)
-  $CTA
+  <% if calculator.are_you_vulnerable == "yes" %>
+    $CTA
+      ## If you are clinically vulnerable: get your employer to make arrangements for you to work safely
+
+      Clinically vulnerable people are at higher risk of severe illness (for example, people with pre-existing conditions). Therefore, you should take extra care in observing social distancing. Your employer should help you to work from home, either in your current role or in an alternative role.
+
+      If you are clinically vulnerable (but not extremely clinically vulnerable) and cannot work from home, your employer should offer you the option of the safest available on-site roles, enabling you to stay 2 metres away from others.
+
+      If you have to spend time within 2 metres of others, employers should carefully assess whether this involves an acceptable level of risk. As for any workplace risk, employers must take into account specific duties to those with protected characteristics, including, for example, pregnant women who are entitled to a suspension on full pay if suitable roles cannot be found.
+
+      Find out more about how to work safely in your workplace.
```

Comment:
Figma would suggest this needs to be a link to https://www.gov.uk/coronavirus-business-reopening